lauraguerra1/whats-cookin
# Adds initial recipe modal skeleton and listeners

## Description
This pr adds the initial skeleton for our clicked recipe view!

## Testing Notes
- pull down changes and `npm start` to start the server
  - navigate to `http://localhost:8080/` to see the local site
- click on any recipe to pull up the recipe view (listener)
- clicking the `X` in the top right closes the modal (listener)
  - make sure the listener only triggers on the actual X and no other part of the page
  - make sure the proper css classes are being added/removed on open and close
